src/App.js

```diff
@@ -1,8 +1,16 @@
 import React from 'react';
 import './App.scss';
+import Sum from './components/Sum';
 
 const App = () => (
-  <h1>React sum</h1>
+  <>
+    <h1 className="title">React sum</h1>
+    <article className="sum">
+      <p className="sum--1"><Sum a={2} b={2} /></p>
```

Comment:
```suggestion
      <p className="sum--1">
        <Sum a={2} b={2} />
      </p>
```
когда компонент внутри другого тега, разносим их по разным строкам

Reply:
ok